src/components/main/atom/CarouselScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledCarouselScreen = styled.div`
+  height: 100%;
+
+  position: relative;
+`;
+const SubCopy = styled.span<{ color: string }>`
+  color: ${(props) => props.color};
+  @media (min-width: 780px) {
+    font-size: 20px;
+  }
+  @media (max-width: 779px) {
+    font-size: 16px;
```

Comment:
이건 각각 안주고 한쪽만 주고 나머지는 default 값으로 설정해줘도될거 같아요.
```suggestion
   font-size: 20px;
  @media (max-width: 779px) {
    font-size: 16px;
```
이든 반대든

Reply:
수정하도록 하겠습니다 :)